### PR TITLE
Nbo/make scenario prameters more accessible

### PR DESCRIPTION
**Pull request by nborde-CSM**
Dev Review rules:
- 3 PR max
- at the second feedback, prefer meet instead of github conversation

Definition of Done:
- [ ] Testing
- [ ] No error logs in web console (except iFrame errors)
- [ ] Create or Update documentation (README file for users/integrators)
- [ ] Check the [test page](https://spaceport.cosmotech.com/confluence/pages/viewpage.action?spaceKey=QA&title=Azure+Sample+Webapp) in order to manually validate the feature 
- [ ] Clean code (regroup configuration variable, typos, no console.log...)

### CODE REVIEW

File: src/components/ScenarioParameters/ScenarioParameters.js [hunk 1]
```diff
@@ -233,50 +263,62 @@ const ScenarioParameters = ({
   const noTabsShown = parametersGroupsMetadata.length === 0;
   const isCurrentScenarioRunning = currentScenario.data.state === SCENARIO_RUN_STATE.RUNNING;
 
+  const [accordionSummaryExpanded, setAccordionSummaryExpanded] = useState(
+    localStorage.getItem('parametersAccordionExpanded') === 'true'
+  );
```
Comment (csm-thu) [inline src/components/ScenarioParameters/ScenarioParameters.js:268]: Could you move the `useState` declaration upward, near the top of the component declaration, to be next to the other `useState`s ?

**Review by csm-thu — CHANGES_REQUESTED** [review]

File: cypress/commons/actions/generic/Scenarios.js [hunk 2]
```diff
@@ -12,9 +12,6 @@ function getScenarioViewTab() {
 function getScenarioView() {
   return cy.get(GENERIC_SELECTORS.scenario.view);
 }
-function getScenarioParametersTab() {
-  return cy.get(GENERIC_SELECTORS.scenario.parameters.tabs);
-}
```
[the remark this answers is not in the document]
Comment (nborde-CSM) [reply cypress/commons/actions/generic/Scenarios.js:17]: I thought I removed it...

File: src/views/Scenario/Scenario.js [hunk 3]
```diff
@@ -254,9 +237,24 @@ const Scenario = (props) => {
                 userRoles={user.roles}
               />
             )}
-          </Card>
+          </Grid>
         </Grid>
       </Grid>
+      <Grid item xs={12}>
+        <Card style={{ height: '400px' }}>
```
[the remark this answers is not in the document]
Comment (nborde-CSM) [reply src/views/Scenario/Scenario.js:244]: It's an old miss...

File: src/components/ScenarioParameters/ScenarioParameters.js [hunk 4]
```diff
@@ -29,6 +33,19 @@ const useStyles = makeStyles((theme) => ({
     alignItems: 'center',
     margin: `0 ${theme.spacing(3)}px`,
   },
+  accordion: {
+    backgroundColor: theme.palette.background.card,
+  },
+  accordionSummary: {
+    flexDirection: 'row-reverse',
```
[the remark this answers is not in the document]
Comment (nborde-CSM) [reply src/components/ScenarioParameters/ScenarioParameters.js:40]: row-reverse is required to set expand/collapse icon on the right side of the accordion summary

File: cypress/commons/actions/generic/Scenarios.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (nborde-CSM) [reply cypress/commons/actions/generic/Scenarios.js:17]: scenario.parameters.tabs is still used with ScenarioParameters
It was just the same function.

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Pull request merged**